src/components/modal/ScriptOntologyModal.jsx

```diff
@@ -30,37 +30,35 @@ class ScriptOntologyModal extends React.Component {
     }
   }
 
-  handleRemoveOntology(ontologyURI) {
-    console.log(this.state.scriptPath + " #### " + ontologyURI);
-    Rest.removeScriptOwnOntology(this.state.scriptPath, ontologyURI).then((response) => {
-      if (response.status === 200) {
-        Rest.getScriptOwnOntology(this.state.scriptPath).then((response2) => {
-          this.setState({
-            scriptPath: this.state.scriptPath,
-            ontologies: response2,
-          });
-        });
-      } else {
-        alert(ontologyURI + " can not be deleted.");
-      }
-    });
+  async handleRemoveOntology(ontologyURI) {
+    try {
+      console.log(this.state.scriptPath + " #### " + ontologyURI);
+      await Rest.removeScriptOwnOntology(this.state.scriptPath, ontologyURI);
+      const response = await Rest.getScriptOwnOntology(this.state.scriptPath);
+      this.setState({
+        scriptPath: this.state.scriptPath,
+        ontologies: response,
+      });
+    } catch (error) {
+      alert(`An error occurred while removing the ontology: ${ontologyURI}. ${error.message || error}`);
```

Comment:
This looks like inconsistent with the rest of what you wrote ... you always logged using
alert() --- which was less specific
console.log() -- which was more specific

Here, it is the opposite way around ... 

Moreover, here you use console.error and elsewhere you use console.log ... 


Is it the way you wanted ?

Reply:
Yes, console.log will provide more detailed information about the error, while alert will only provide basic UI information.

About logs, I think a better solution would be to use console.error for errors, as it helps differentiate between logs related to states and actual error logs.